radicle-node/src/worker.rs

```diff
@@ -190,70 +185,13 @@ impl<G: Signer + Ecdh + 'static> Worker<G> {
-
-                        // Set the read timeout for the `done` packet to twice the configured
-                        // value that is used for the fetching (initiator) side.
-                        //
-                        // This is because the uploader always waits for the `done` packet;
-                        // so in case the fetch is aborted by the uploader, eg. if
-                        // it can't connect with the daemon, it will wait long enough for the
-                        // fetcher to timeout before timing out itself, and will thus receive
-                        // the `done` packet.
-                        pktline_r
-                            .stream()
-                            .as_connection()
-                            .set_read_timeout(Some(self.timeout * 2))
-                            .ok();
-
-                        loop {
-                            match pktline_r.read_done_pktline(&mut header) {
-                                Ok(()) => {
-                                    log::debug!(target: "worker", "Received `done` packet from remote");
-
-                                    // If we get the `done` packet, we exit with the original
-                                    // error.
-                                    return (
-                                        WireSession::from_split_io(stream_r, stream_w),
-                                        Err(err.into()),
-                                    );
-                                }
-                                Err(e) if e.kind() == io::ErrorKind::InvalidInput => {
-                                    // If we get some other packet, because the fetch request
-                                    // is still sending stuff, we simply keep reading until we
-                                    // get a `done` packet.
-                                    continue;
-                                }
-                                Err(_) => {
-                                    // If we get any other error, eg. a timeout, we abort.
-                                    log::error!(
-                                        target: "worker",
-                                        "Upload of {} to {} aborted: missing `done` packet from remote",
-                                        fetch.rid,
-                                        fetch.remote
-                                    );
-                                    return (
-                                        WireSession::from_split_io(stream_r, stream_w),
-                                        Err(FetchError::RemoteAbortedFetch),
-                                    );
-                                }
-                            }
-                        }
-                    }
-                }
+                // XXX: two fetches are expected to happen, one for
+                // the `rad` refs, followed by the refs listed in
+                // signed refs.
```

Comment:
`XXX` is basically an `Nb.` or a heads up. So it's like, "Hey, be careful if you're changing this because of the expected two fetched".

Not necessarily a TODO, but possibly something that can be improved in the future :)

Reply:
Oh, then please change it to `Nb.`, because that's what we use in the codebase :)